src/H5Zzfp.c

```diff
@@ -48,9 +47,23 @@ and calls to bitstream methods with 'B ' as in
 #include "H5Zzfp_props_private.h"
 
 /* Convenient CPP logic to capture ZFP lib version numbers as compile time hex number */
-#define ZFP_VERSION_NO__(Maj,Min,Rel)  (0x0 ## Maj ## Min ## Rel)
-#define ZFP_VERSION_NO_(Maj,Min,Rel)   ZFP_VERSION_NO__(Maj,Min,Rel)
-#define ZFP_VERSION_NO                 ZFP_VERSION_NO_(ZFP_VERSION_MAJOR,ZFP_VERSION_MINOR,ZFP_VERSION_RELEASE)
+#define ZFP_VERSION_NO__(Maj,Min,Pat,Twk)  (0x ## Maj ## Min ## Pat ## Twk)
+#define ZFP_VERSION_NO_(Maj,Min,Pat,Twk)   ZFP_VERSION_NO__(Maj,Min,Pat,Twk)
+#if defined(ZFP_VERSION_RELEASE)
+#define ZFP_VERSION_NO                 ZFP_VERSION_NO_(ZFP_VERSION_MAJOR,ZFP_VERSION_MINOR,ZFP_VERSION_RELEASE,0)
+#elif defined(ZFP_VERSION_PATCH)
+#define ZFP_VERSION_NO                 ZFP_VERSION_NO_(ZFP_VERSION_MAJOR,ZFP_VERSION_MINOR,ZFP_VERSION_PATCH,0)
+#elif defined(ZFP_VERSION_PATCH)
```

Comment:
I have the impression that this second "#elif defined(ZFP_VERSION_PATCH) will never be triggered.